packages/search-in-workspace/src/browser/styles/index.css

```diff
@@ -255,6 +255,7 @@
     background: var(--theia-word-highlight-match-color1);
     display: inline-block;
     line-height: normal;
+    white-space: pre;
```

Comment:
<img width="586" alt="Screen Shot 2019-08-20 at 13 26 09" src="https://user-images.githubusercontent.com/914497/63343389-1a712980-c34e-11e9-904d-f2e11c7bee59.png">

`pre` seems to include a visual leading space.

Reply:
I think there is an extra space somehow in the widget itself, I'll remove it since its unecessary.
Thank you for pointing it out @AlexTugarev.

Reply:
Yeah! you're right, looking at the DOM `<div class="resultLine noWrapInfo " title="testblablub bla"><span>test</span><span class="match"> blablub </span><span>bla</span></div>` there is a space inserted which is definitely not part of the search string.

Would be awesome if you can sort it out here 🙏

Reply:
I removed the unnecessary space in the widget, nice catch! :)

<div align='center'>

![image](https://user-images.githubusercontent.com/40359487/63343820-ecbfbc80-c31c-11e9-8e43-f2d585d600aa.png)


</div>